identity/registry/provider_registrar_test.go

```diff
@@ -222,3 +227,16 @@ func (mt *mockTransactor) RegisterIdentity(id string, stake, fee *big.Int, benef
 func (mt *mockTransactor) CheckIfRegistrationBountyEligible(identity identity.Identity) (bool, error) {
```

Comment:
Had we already existing such function before (not mock)? Can't find one in this PR.

Reply:
Yes we do - for free registration (with tokens i guess). But it's enabled for Betanet, but not Testnet2